.gitignore

```diff
@@ -1,4 +1,7 @@
 # local development
 /config-local
 # Maven
-**/target/
+**target/
+**.settings
```

Comment:
Typically .gitignore should only contain entries which are needed by everybody.
All other entries can either go into your global git settings, or you can add them
to your local project settings in the file .git/info/exclude.